.github/SECURITY.md

```diff
@@ -0,0 +1,23 @@
+<h2>Responsible disclosure policy</h2>
```

Comment:
Any specific reason why this is HTML and not markdown ?

Reply:
No specific reason.

The security guys wrote that and I just kept it the same.